projects/packages/forms/src/contact-form/class-contact-form-field.php

```diff
@@ -674,11 +674,11 @@ public function render_radio_field( $id, $label, $value, $class, $required, $req
 			$option = Contact_Form_Plugin::strip_tags( $option );
 			if ( is_string( $option ) && $option !== '' ) {
 				$radio_value = $this->get_option_value( $this->get_attribute( 'values' ), $option_index, $option );
-				$radio_id    = "$id-$radio_value";
+				$radio_id    = sanitize_html_class( $id ) . '-' . sanitize_html_class( $radio_value );
```

Comment:
I don't think we need to wrap `$id` with sanitize_html_class( $id ). Since if this is the case we would need to do it in a few different places as well see line 668.

Reply:
Well, if there's any chance that $id could contain non-valid characters for an html id, then the fact that we need to wrap it multiple times would be ok, and the right thing to do. 

But we create that id ourselves, and I believe it should be all valid characters, so you're right this is probably overkill for that reason.